purchase_work_acceptance_invoice_plan/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase Work Acceptance Invoice Plan",
+    "version": "13.0.1.0.0",
+    "category": "Purchase Management",
+    "author": "Ecosoft, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "depends": ["purchase_work_acceptance", "purchase_invoice_plan"],
+    "data": [
+        "views/purchase_views.xml",
+        "views/work_acceptance_view.xml",
+        "wizard/select_work_acceptance_invoice_plan_wizard_views.xml",
+    ],
+    "maintainer": [""],
+    "installable": True,
```

Comment:
```suggestion
    "installable": True,
    "auto_install": True,
```
Because we want this module to install when both `purchase_invoice_plan` and `purchase_work_acceptance` is installed.